layers/meta-resin-raspberrypi/recipes-support/resin-init/files/resin-init-board

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
```

Comment:
It seems to be customary in scripts. On the third line there would usually be a description or licence. In this case there is no comment, but I felt this looks better for consistency. I can remove it if it's ugly.